### ISSUE TITLE
Compatibility issue with php-ffmpeg ^0.8 while installing 

### ISSUE THREAD

**Issue by Chinese1904** [pascalbaljetmedia/laravel-ffmpeg#3]
I started with a fresh Laravel install, added the required packages through composer and then tried to `composer require pbmedia/laravel-ffmpeg`.

The error message I get is
```
Using version ^1.1 for pbmedia/laravel-ffmpeg
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - pbmedia/laravel-ffmpeg 1.1.3 requires php-ffmpeg/php-ffmpeg ^0.6 -> satisfiable by php-ffmpeg/php-ffmpeg[0.6.0, 0.6.1] but these conflict with your requirements or minimum-stability.
    - pbmedia/laravel-ffmpeg 1.1.2 requires php-ffmpeg/php-ffmpeg ^0.6 -> satisfiable by php-ffmpeg/php-ffmpeg[0.6.0, 0.6.1] but these conflict with your requirements or minimum-stability.
    - pbmedia/laravel-ffmpeg 1.1.1 requires php-ffmpeg/php-ffmpeg ^0.6 -> satisfiable by php-ffmpeg/php-ffmpeg[0.6.0, 0.6.1] but these conflict with your requirements or minimum-stability.
    - pbmedia/laravel-ffmpeg 1.1.0 requires php-ffmpeg/php-ffmpeg ^0.6 -> satisfiable by php-ffmpeg/php-ffmpeg[0.6.0, 0.6.1] but these conflict with your requirements or minimum-stability.
    - Installation request for pbmedia/laravel-ffmpeg ^1.1 -> satisfiable by pbmedia/laravel-ffmpeg[1.1.0, 1.1.1, 1.1.2, 1.1.3].
```

As you can see in my composer.json file, everything seems to be included right.
```
"require": {
        "php": ">=5.6.4",
        "intervention/image": "^2.3",
        "laracasts/flash": "^2.0",
        "laravel/framework": "5.4.*",
        "laravel/socialite": "^3.0",
        "laravel/tinker": "~1.0",
        "laravelcollective/html": "5.*",
        "league/flysystem": "^1.0",
        "php-ffmpeg/php-ffmpeg": "^0.8.0",
        "phpoffice/phpexcel": "1.8.1",
        "rtconner/laravel-likeable": "~1.2",
        "rtconner/laravel-tagging": "~2.2",
        "yajra/laravel-datatables-oracle": "~6.0"
    },
```

Any help would be highly appreciated!

**Comment by jamesgraham** [comment]
@pascalbaljet can you please consider updating your php-ffmpeg version in composer to support this?

>        "php-ffmpeg/php-ffmpeg": "^0.8",

**Comment by Chinese1904** [comment]
would be highly appreciated 👍

**Comment by pascalbaljet** [comment]
Should be fixed in 1.1.4! Thanks for notifying me :)
